test/integration/edition_edit_test.rb

```diff
@@ -575,6 +605,99 @@ class EditionEditTest < IntegrationTest
     end
   end
 
+  context "Tag organisation page" do
```

Comment:
Very minor, but should this be be plural?
```suggestion
  context "Tag organisations page" do
```